.changeset/slimy-sloths-nail.md

```diff
@@ -0,0 +1,18 @@
+---
+'@shopify/cli-hydrogen': minor
+---
+
+The `deploy` command now displays an error if there are uncommited changes in a project's Git repository. If you'd like to go ahead with the deployment regardless, you can use the new `force` flag.
```

Comment:
I guess in a public repo, it is difficult to hide things. Hopefully we can go public soon.

Reply:
Yeah that's what I meant, we don't need to hide the code but it might be confusing if it's part of the release notes but still not usable 🤔 
Should we then just remove this changelog for now?